source/release_notes/4.4.16-16.rst

```diff
@@ -0,0 +1,51 @@
+.. _PSMDB-4.4.16-16:
+
+================================================================================
+*Percona Server for MongoDB* 4.4.16-16 (2022-08-30)
+================================================================================
+
+:Date: August 30, 2022
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.4/install/index.html>`_
+
+Percona Server for MongoDB 4.4.16-16 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.4.16 Community Edition.
+It supports MongoDB 4.4.16 protocols and drivers.
+
+Release Highlights
+==================
+
+* `SERVER-67302 <https://jira.mongodb.org/browse/SERVER-67302>`_ - Fixed the server crash with the CLOCK_REALTIME set to forward by making the linearizable reads robust to primary catch-up and simultaneous stepdown.
+* `SERVER-61321 <https://jira.mongodb.org/browse/SERVER-61321>`_ - Improved handling of large/NaN (Not a Number) values for text index version.
+* `SERVER-60607 <https://jira.mongodb.org/browse/SERVER-60607>`_ - Improved handling of large/NaN (Not a Number) values for geo index version.
+* `SERVER-66418 <https://jira.mongodb.org/browse/SERVER-66418>`_ -  Fixed the issue with bad projection created during dependency analysis due to string order assumption. It resulted in the ``PathCollision`` error. The issue is fixed by improving dependency analysis for projections by folding dependencies into ancestor dependencies where possible.
+* `WT-9096 <https://jira.mongodb.org/browse/WT-9096>`_ - Fixed the wrong key/value returning during search near when the key doesn't exist.
+  
+Find the full list of changes in the `MongoDB 4.4.16 Community Edition release notes <https://www.mongodb.com/docs/v4.4/release-notes/4.4/#4.4.16---aug-19--2022>`_.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-1046`: Make the ``kmipKeyIdentifier`` option not mandatory
+  
+.. note::
+
+   If you have configured data at rest encryption and wish to upgrade |PSMDB|, go through the `encrypting existing data steps <https://www.mongodb.com/docs/v4.4/tutorial/configure-encryption/#std-label-encrypt-existing-data>`_ during the upgrade as follows: 
```

Comment:
This is only true for cases data-at-rest + KMIP feature (which is still technical preview), so I'd re-phase:
If you have configured data at rest encryption with KMIP server*

Reply:
Updated